Cubical/Foundations/HLevels.agda

```diff
@@ -22,14 +22,16 @@ open import Cubical.Foundations.Transport
 open import Cubical.Foundations.HAEquiv      using (congEquiv)
 open import Cubical.Foundations.Univalence   using (ua; univalence)
 
-open import Cubical.Data.Sigma using (ΣPathP; sigmaPath→pathSigma; pathSigma≡sigmaPath; _Σ≡T_)
+open import Cubical.Data.Sigma using (pathSigma≡sigmaPath; _Σ≡T_; ΣProp≡)
 open import Cubical.Data.Nat   using (ℕ; zero; suc; _+_; +-zero; +-comm)
 
 private
   variable
-    ℓ ℓ' : Level
+    ℓ ℓ' ℓ'' ℓ''' : Level
     A : Type ℓ
     B : A → Type ℓ
+    C : (x : A) → B x → Type ℓ''
+    D : (x : A) (y : B x) → C x y → Type ℓ''
```

Comment:
I'm pretty sure you could just use `ℓ` for `C` and `D` too?

Reply:
Oh, I intended to use l''' for D. But maybe that's not needed? Will Agda maximally generalize everything?

Reply:
I'm pretty sure it will. the `ℓ` in the types of `A` and `B` are not constrained to be the same either.

You only need two level `variable`s for something like `F : Set l -> Set l'` where you actually write both in the same type.

Reply:
Ah right, I keep forgetting how variables work in Agda... It's so confusing compared to Coq. Will fix this